README.md

```diff
@@ -0,0 +1,19 @@
+# Glossaico
```

Comment:
FYI it's highly unlikely any user will ever see this. It's better to include anything user-facing in appdata/metainfo.

Reply:
So, you suggest that I remove the README file?